bq-workers/argocd-parser/main.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import base64
+import os
+import json
+
+import shared
+
+from flask import Flask, request
+
+app = Flask(__name__)
+
+
+@app.route("/", methods=["POST"])
+def index():
+    """
+    Receives messages from a push subscription from Pub/Sub.
+    Parses the message, and inserts it into BigQuery.
+    """
+    event = None
+    envelope = request.get_json()
```

Comment:
```suggestion
    if not request.is_json:
        raise Exception("Expecting JSON payload")
    envelope = request.get_json()
```

Reply:
Updated the way the envelope is handled! See example here -- https://github.com/GoogleCloudPlatform/fourkeys/blob/main/bq-workers/circleci-parser/main.py#L34

Reply:
Great, thanks. 🙂